plugin/ochttp/client.go

```diff
@@ -51,6 +52,11 @@ type Transport struct {
 	// name equals the URL Path.
 	FormatSpanName func(*http.Request) string
 
+	// NewClientTrace may be set to a function allowing the current *trace.Span
+	// to be annotated with HTTP request event information emitted by the
+	// httptrace package.
+	NewClientTrace func(*http.Request, *trace.Span) *httptrace.ClientTrace
```

Comment:
We don't often call fields after New* in Go. I'd rather wished this was named ClientTrace. Can we make this modification given there are no users of this yet?

Reply:
Not a huge deal if we are not doing it. I also don't think sync.Pool.New is well representative of the overall Go style in these cases.

Reply:
Do you have an example from the standard library of something that you think is representative? I couldn't think of anything except that.

I think it's fine to make changes since we haven't created a release yet.